onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -242,7 +242,11 @@ def main():
     args = parser.parse_args()
     calibrate_op_types = args.op_types.split(',')
     black_nodes = args.black_nodes.split(',')
+    while '' in black_nodes:
```

Comment:
`black_nodes=[x for x in black_nodes if x]` and `black_nodes=filter(None, black_nodes)` are more pythonic, being the latter faster

Reply:
Cool! Pythonic!